README.md

```
@@ -1,6 +1,6 @@
 
 # CredChain: Decentralized Credentials Verification
-  
+  #
```

markdownlint-cli2 (0.17.2)

Headings must start at the beginning of the line

(MD023, heading-start-left)